test/test_scanner.py

```diff
@@ -43,7 +43,6 @@
                 prod_list.append(i["product"])
         for i in prod_list:
             all_the_tests.append(i)
-all_the_tests.append("kerberos")
```

Comment:
I asked about this in code review and I gather we need this because kerberos was missing some test data and wasn't getting automatically appended.  Since I can't easily edit deleted lines from the code review interface, I think instead I'll open up a new issue to fix whatever's up with kerberos.